pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -199,6 +199,7 @@ stdenv.mkDerivation rec {
         "S3RegionResolutionTest.PublicBucket"
         "S3RegionResolutionTest.RestrictedBucket"
         "TestMinioServer.Connect"
+        "TestS3FS.DeleteDir"
```

Comment:
Strangely, I get a test failure with this:
```
[==========] Running 20 tests from 3 test suites.
[----------] Global test environment set-up.
[----------] 2 tests from S3OptionsTest
[ RUN      ] S3OptionsTest.FromAccessKey
[       OK ] S3OptionsTest.FromAccessKey (0 ms)
[ RUN      ] S3OptionsTest.FromAssumeRole
[       OK ] S3OptionsTest.FromAssumeRole (2 ms)
[----------] 2 tests from S3OptionsTest (3 ms total)

[----------] 2 tests from S3FileSystemRegionTest
[ RUN      ] S3FileSystemRegionTest.Default
[       OK ] S3FileSystemRegionTest.Default (2 ms)
[ RUN      ] S3FileSystemRegionTest.EnvironmentVariable
[       OK ] S3FileSystemRegionTest.EnvironmentVariable (0 ms)
[----------] 2 tests from S3FileSystemRegionTest (2 ms total)

[----------] 16 tests from TestS3FS
[ RUN      ] TestS3FS.GetFileInfoBucket
-- Arrow Fatal Error --
IOError: bind() failed: Operation not permitted
/private/tmp/nix-build-arrow-cpp-6.0.1.drv-0/apache-arrow-6.0.1/cpp/build/src/arrow/filesystem
```
This is confusing – I believe all tests used to run in darwin sandbox before

Reply:
I wonder if they should've been passing at all, since these tests access the network IIRC.